libraries/AP_HAL_ChibiOS/hwdef/PodmanH7/hwdef.dat

```diff
@@ -0,0 +1,24 @@
+# hw definition file for processing by chibios_hwdef.py
+# for PodmanH7 hardware.
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1120
+
+include hwdef.inc
+
+# List of files to put in ROMFS. For fmuv3 we need an IO firmware so
+# we can automatically update the IOMCU firmware on boot. The format
+# is "ROMFS ROMFS-filename source-filename". Paths are relative to the
+# ardupilot root.
+ROMFS io_firmware.bin Tools/IO_Firmware/iofirmware_f103_8MHz_lowpolh.bin
```

Comment:
if you have an f103 and don't have level shifters then you can also support the iomcu bdshot feature. Please see the Pixhawk6X hwdef for an example

Reply:
It requires providing an extra iomcu firmware

Reply:
Thank you.
I have an f103 with 8Mhz IOMCU.

Reply:
Pixhawk6X have an f103 with 24Mhz IOMCU.
I need to build iofirmware_f103_8MHz_dshot_highpolh iomcu firmware.

Reply:
> if you have an f103 and don't have level shifters then you can also support the iomcu bdshot feature. Please see the Pixhawk6X hwdef for an example

My local testing on this computer has resulted in no issues.
I must wait for Pull Request (PR) number 27087 to merge into the master branch before making changes; otherwise, the compilation will fail.

To integrate this code snippet correctly, it should be appended at the end of the **hwdef.dat** file.

      # enable support for dshot on iomcu
      ROMFS io_firmware_dshot.bin Tools/IO_Firmware/iofirmware_f103_8MHz_dshot_lowpolh.bin
      define HAL_WITH_IO_MCU_BIDIR_DSHOT 1